scripts/openmc-make-depletion-chain-casl

```diff
@@ -0,0 +1,240 @@
+    neutron_files = glob.glob(os.path.join(endf_dir, 'neutrons', '*.endf'))
+
+    # Create a Chain
+    chain = openmc.deplete.Chain()
+
+    print('Reading ENDF nuclear data from "{}"...'.format(os.path.abspath(endf_dir)))
+
+    # Create dictionary mapping target to filename
+    print('Processing neutron sub-library files...')
+    reactions = {}
+    for f in neutron_files:
+        evaluation = openmc.data.endf.Evaluation(f)
+        name = evaluation.gnd_name
+        if name in CASL_CHAIN:
+            reactions[name] = {}
+            for mf, mt, nc, mod in evaluation.reaction_list:
+                if mf == 3:
+                    file_obj = StringIO(evaluation.section[3, mt])
+                    openmc.data.endf.get_head_record(file_obj)
+                    q_value = openmc.data.endf.get_cont_record(file_obj)[1]
+                    reactions[name][mt] = q_value
+
+    # Determine what decay and FPY nuclides are available
+    print('Processing decay sub-library files...')
+    decay_data = {}
+    for f in decay_files:
+        data = openmc.data.Decay(f)
+        name = data.nuclide['name']
+        if name in CASL_CHAIN:
+            decay_data[name] = data
+
+    for name in CASL_CHAIN:
+        if name not in decay_data:
+            print('WARNING: {} has no decay data!'.format(name))
+
+    print('Processing fission product yield sub-library files...')
+    fpy_data = {}
+    for f in fpy_files:
+        data = openmc.data.FissionProductYields(f)
+        name = data.nuclide['name']
+        if name in CASL_CHAIN:
+            fpy_data[name] = data
+
+    print('Creating depletion_chain...')
+    missing_daughter = []
+    missing_rx_product = []
+    missing_fpy = []
+
+    for idx, parent in enumerate(sorted(decay_data, key=openmc.data.zam)):
+        data = decay_data[parent]
+
+        nuclide = Nuclide()
+        nuclide.name = parent
+
+        chain.nuclides.append(nuclide)
+        chain.nuclide_dict[parent] = idx
+
+        if not CASL_CHAIN[parent][0] and \
+           not data.nuclide['stable'] and data.half_life.nominal_value != 0.0:
+            nuclide.half_life = data.half_life.nominal_value
+            nuclide.decay_energy = sum(E.nominal_value for E in
+                                       data.average_energies.values())
+            sum_br = 0.0
+            for i, mode in enumerate(data.modes):
+                type_ = ','.join(mode.modes)
+                if mode.daughter in decay_data:
+                    target = mode.daughter
+                else:
+                    print('missing {} {} {}'.format(parent, ','.join(mode.modes), mode.daughter))
+                    continue
+
+                # Write branching ratio, taking care to ensure sum is unity
+                br = mode.branching_ratio.nominal_value
+                sum_br += br
+                if i == len(data.modes) - 1 and sum_br != 1.0:
+                    br = 1.0 - sum(m.branching_ratio.nominal_value
```

Comment:
Question: If the branching ratio doesn't sum to unity, is this operation rescaling them to unity? I'm basing this off my very basic understanding of how I would expect these calculations to happen under the standard model. Or is this another normalization operation?

If this is the case, for readability, I would clarify that you are rescaling them (rather than ensuring summation) in the comment on line 118.